src/Migrations/Version20230523154501.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * MAT-300 – Include some CC22 donations for belated, internal Gift Aid claims.
+ */
+final class Version20230523154501 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Include some CC22 donations for Gift Aid claims';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(<<<SQL
+            UPDATE Donation SET tbgShouldProcessGiftAid = 1
+            WHERE campaign_id = 4216 AND
```

Comment:
Yes, I've checked in the DB, this is the ID of the most recent campaign for the relevant charity.